exchanges/lbank/lbank_types.go

```diff
@@ -26,9 +26,9 @@ type TickerResponse struct {
 // MarketDepthResponse stores arrays for asks, bids and a timestamp for a currecy pair
 type MarketDepthResponse struct {
 	ErrCapture `json:",omitempty"`
-	Asks       [][]float64 `json:"asks"`
-	Bids       [][]float64 `json:"bids"`
-	Timestamp  int64       `json:"timestamp"`
+	Asks       [][]string `json:"asks"`
+	Bids       [][]string `json:"bids"`
```

Comment:
you can add to the json tag so as not to use the strconv.ParseFloat in lbank_wrapper.go. Example:
**Asks       [][]float64`json:"asks,string"`**

Reply:
Sorry, this is incorrect.